### DISCUSSION TITLE
More language converters

### DISCUSSION THREAD

**Discussion by kobi2187** [vlang/v#15873] [Ideas]
Hi, I am new here. V seems to be a quite nice language. 
I think to build a vast stdlib quickly, you should find ways to port existing mature libraries from other languages. 
I don't know which is the most semantically similar to V, but if you find a few, and build automatic converters, this is a greatly accelerating factor. 
Especially since many libs written from scratch tend to fall into the same bugs, and a mature library can at least give you the unit tests, if not the actual "battle-tested" code.
in the past I wrote a c# to nim converter, but semantic equivalence was not perfect, and it's also a much larger project than it seems at first - but not impossible, at all. 
I don't have any authority, just wanted to suggest that I think in this modern age, this is the way to go. convert existing code by automatic porting, and invest in such tools.

**Comment by larpon** [comment]
Please see:
https://github.com/vlang/c2v
https://github.com/vlang/go2v
https://github.com/vlang/py2v

**Reply by ghost** [reply]
also interesting project https://github.com/ChAoSUnItY/v-jvm
